packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.stories.ts

```diff
@@ -0,0 +1,120 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { withActions } from '@storybook/addon-actions/decorator';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../../utilities/tests/storybook';
+import { tableTag } from '../../../table';
+import { tableColumnDateTextTag } from '..';
+import {
+    SharedTableArgs,
+    sharedTableActions,
+    sharedTableArgTypes,
+    sharedTableArgs
+} from '../../base/tests/table-column-stories-utils';
+import { tableColumnTextTag } from '../../text';
+
+const simpleData = [
+    {
```

Comment:
nit / optional / friday afternoon:
Some simpsons birthdays ([floating timeline 2023](https://simpsons.fandom.com/wiki/Floating_Timeline#With_Floating_Timeline_(2023))):
Patty and Selma Bouvier (twins) - June 11, 1979
Maggie Simpson - January 12, 2022
Homer Simpson - May 12, 1984
Marge Simpson - March 19, 1984
Bart Simpson - April 1, 2013
Lisa Simpson - May 9, 2015

Reply:
I had considered that, but realized these characters have perpetually changing birthdays. Anyway, I've changed them to birthdays (but left the random times).